src/pool-service/IrcClientRedisState.ts

```diff
@@ -21,20 +36,50 @@ interface IrcClientStateDehydrated {
     capabilities: ReturnType<IrcCapabilities["serialise"]>;
     supportedState: IrcSupported;
```

Comment:
Let's make this optional to let type hints enforce the possibility of serialization errors:
```suggestion
    supportedState?: IrcSupported;
```